element-hq/element-android
# Fix Alert verification layout text in RTL devices

## Fix Alert verification layout text in RTL devices



## Screenshots / GIFs


## Before

![Untitled](https://github.com/element-hq/element-android/assets/16717834/a041c8e1-cda0-45e2-9c45-1689d4439de5)


## After

![Untitled2](https://github.com/element-hq/element-android/assets/16717834/766ba4e2-f36c-452f-8a84-c241f99c5689)



## Tests


- Step 1: Open the settings page.
- Step 2: Open Preferences.
- Step 3: Change the language to "فارسی - عربی".
- Step 4
- Open home page
- Step 5
- See top of screeen


## Tested devices

- OS version(s):

## Checklist
